packages/e2e-tests/authenticated/new-tests-suite/tests-01.test.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  TEST_RUN_WORKSPACE_API_KEY,
+  TEST_RUN_WORKSPACE_TEAM_ID,
+} from "../../helpers/authentication";
+import { openContextMenu } from "../../helpers/console-panel";
+import { selectContextMenuItem } from "../../helpers/context-menu";
+import test, { expect } from "../../testFixtureTestRuns";
+import {
+  filterTestsByText,
+  findTestByText,
+  noTestMatches,
+  noTestSelected,
+  sortTestsBy,
+  startTest,
+  testsItems,
+  waitForTestExecutions,
+} from "./test-suite.utils";
+
+test.use({ testRunState: "UNIQUE_TESTS_FOR_TESTS_VIEW" });
+
+test(`authenticated/new-test-suites/tests-01: basic tests`, async ({ pageWithMeta: { page } }) => {
+  await startTest(page, TEST_RUN_WORKSPACE_API_KEY, TEST_RUN_WORKSPACE_TEAM_ID);
+  expect(await testsItems(page).count()).not.toBe(0);
+
+  //#region > Test runs list
+  const rateSortDropdown = page.locator('[data-test-id="TestPage-ResultFilter-DropdownTrigger"]');
+
+  //#region >>> Each test should be displayed in the first column by title
+  expect(await testsItems(page).count()).not.toBe(0);
+  await filterTestsByText(page, "cypress-spec-uniq.ts");
+  expect(await testsItems(page).count()).toBe(1);
+  expect(await testsItems(page).first().locator('[data-test-id="Test-Title"]').innerText()).toEqual(
+    "cypress-spec-uniq.ts"
+  );
+
+  await filterTestsByText(page, "Tests View -");
+  expect(await testsItems(page).count()).toBe(6);
+  //#endregion
+
+  //#region >>> Sort by flaky rate
+  await sortTestsBy(page, "flakyRate");
+  expect(await testsItems(page, "failure").count()).toBe(0);
+  expect(await testsItems(page, "flaky").count()).not.toBe(0);
+  //#endregion
+
+  //#region >>> Sort by failure rate
+  openContextMenu(rateSortDropdown, { useLeftClick: true });
+  await selectContextMenuItem(page, {
+    contextMenuItemTestId: "failureRate",
+  });
+  expect(await testsItems(page, "failure").count()).not.toBe(0);
+  expect(await testsItems(page, "flaky").count()).toBe(0);
+  //#endregion
+
+  //#region >>> Sort alphabetically
+  await openContextMenu(rateSortDropdown, { useLeftClick: true });
+  await selectContextMenuItem(page, {
+    contextMenuItemTestId: "alphabetical",
+  });
+  expect(await testsItems(page, "failure").count()).not.toBe(0);
+  //TODO: Why is this flaky? It returns "First test", "Fifth test" sometimes
+  expect(await testsItems(page).first().innerText()).toContain("Tests View - Fifth test");
```

Comment:
Our test workspace have mix of both grouped cypress and without group test, and for some reason sorting is broken here. It lists ungrouped tests first then grouped. Not really possible in real workspace though.